README.md

```diff
@@ -5,6 +5,50 @@ Tools for building Pyodide.
 See [http://github.com/pyodide/pyodide](http://github.com/pyodide/pyodide) for
 more information.
 
+## Quickstart
+
+### Usage
+
+> [!TIP]
+> Currently, installing `pyodide-build` is preferred to running it.
+
+#### Install it
+
+```bash
+pip install pyodide-build
+pyodide --help
+pyodide build --help
+```
+
+#### Install it globally
+
+```bash
+pipx install --include-deps pyodide-build
+pyodide --help
+pyodide build --help
+```
+
+or, alternatively
+
+```bash
+pipx install pyodide-cli
+pipx inject pyodide-cli pyodide-build
+pyodide --help
+pyodide build --help
+```
+
+> [!NOTE]
+> If installing `pyodide-build` and its dependencies in virtual environments, we recommend
+using ones that are managed by `venv` or `virtualenv` at the moment; see #58.
```

Comment:
```suggestion
> [!NOTE]
> Currently `pyodide-build` does not work well in environments managed by uv, see #58.
```
I think this is the only thing in this PR that is appropriate for a readme. I think this adds a lot of noise, people can use pipx or uvx or venv or install pyodide-build globally or use whatever tools as they prefer. The readme isn't a tutorial on Python installer workflows.

Reply:
Makes sense to me, thanks!